packages/components/src/DateTimePickers/pickers/DatePicker/DatePicker.scss

```diff
@@ -19,20 +18,34 @@
         align-content: center;
     }
 
-    .calendar-header-row {
-        color: $gray;
+    .calendar-header {
+        .calendar-header-row {
+            color: $gray;
 
-        abbr[title] {
-           font-variant: none;
-           border: none;
-           text-decoration: none;
+            abbr[title] {
+               font-variant: none;
+               border: none;
+               text-decoration: none;
+            }
+        }
+
+        .separator {
+            width: calc(100% - 2rem);
+            border: none;
+            border-bottom: $border-default;
+            margin: 0 auto;
         }
     }
 
-    .separator {
-        width: calc(100% - 2rem);
-        border: none;
-        border-bottom: $border-default;
-        margin: 0 auto;
+
+    .calendar-body {
+        display: flex;
+        flex-direction: column;
+        justify-content: space-evenly;
+        flex: 1;
+
+        .calendar-row {
+            flex: 1;
+        }
```

Comment:
with css module you really don t need cascading, one level is enough, two is ok, three is too much deep.

Reply:
✔